lib/itamae/backend.rb

```diff
@@ -257,6 +257,8 @@ def ssh_options
           password = STDIN.noecho(&:gets).strip
           print "\n"
           opts.merge!(password: password)
+        else
+          opts.merge!(keys_only: true)
```

Comment:
Hi, @ryotarai, this is going to work when setting the key via command line, but is going to fail when the key is set by ssh config. Is this the intended behaviour?